src/include/processor/operator/scan/scan_node_table.h

```diff
@@ -29,26 +29,27 @@ struct ScanNodeTableInfo {
     storage::NodeTable* table;
     std::vector<common::column_id_t> columnIDs;
 
+    std::unique_ptr<storage::NodeTableScanState> localScanState;
+
     ScanNodeTableInfo(storage::NodeTable* table, std::vector<common::column_id_t> columnIDs)
         : table{table}, columnIDs{std::move(columnIDs)} {}
+    EXPLICIT_COPY_DEFAULT_MOVE(ScanNodeTableInfo);
+
+private:
     ScanNodeTableInfo(const ScanNodeTableInfo& other)
         : table{other.table}, columnIDs{other.columnIDs} {}
-
-    std::unique_ptr<ScanNodeTableInfo> copy() const {
-        return std::make_unique<ScanNodeTableInfo>(*this);
-    }
 };
 
 class ScanNodeTable final : public ScanTable {
+    static constexpr PhysicalOperatorType type_ = PhysicalOperatorType::SCAN_NODE_TABLE;
```

Comment:
Why make this a constexpr? Is this used anywhere else other than the constructor?

Reply:
This is a new style I'm trying to enforce for constant values that is local to a class.